src/software/ai/hl/stp/play/ball_placement/ball_placement_play_fsm.cpp

```diff
@@ -101,32 +148,55 @@ void BallPlacementPlayFSM::startWait(const Update &event)
     start_time = event.common.world_ptr->getMostRecentTimestamp();
 }
 
-void BallPlacementPlayFSM::retreat(const Update &event)
+void BallPlacementPlayFSM::releaseBall(const Update &event)
 {
+    PriorityTacticVector tactics_to_run = {{}};
+
     WorldPtr world_ptr = event.common.world_ptr;
     std::optional<Robot> nearest_robot =
         world_ptr->friendlyTeam().getNearestRobot(world_ptr->ball().position());
 
     if (nearest_robot.has_value())
     {
-        PriorityTacticVector tactics_to_run = {{}};
+        wait_tactic->updateControlParams(
+            nearest_robot->position(), nearest_robot->orientation(),
+            TbotsProto::DribblerMode::RELEASE_BALL_SLOW,
+            TbotsProto::BallCollisionType::ALLOW, {AutoChipOrKickMode::OFF, 0},
+            TbotsProto::MaxAllowedSpeedMode::BALL_PLACEMENT_RETREAT,
+            TbotsProto::ObstacleAvoidanceMode::AGGRESSIVE);
+
+        tactics_to_run[0].push_back(wait_tactic);
+    }
+
+    setupMoveTactics(event, event.common.num_tactics -
```

Comment:
needs safety checks that event.common.num_tactics > tactics_to_run[0].size(). Otherwise, we risk underflowing into an unsigned type.